ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPageFactory.java

```diff
@@ -0,0 +1,22 @@
+package io.syndesis.qe.pages.integrations.detail;
+
+public class IntegrationDetailPageFactory {
+
+	public IntegrationDetailPage getDetailPage(String integratinStatus) {
```

Comment:
Typo in `integratinStatus`. But this is interesting approach actually, I need to think about it a bit :)